crates/database/src/orders.rs

```diff
@@ -687,43 +687,22 @@ pub fn solvable_orders(
     sqlx::query_as(OPEN_ORDERS).bind(min_valid_to).fetch(ex)
 }
 
-pub fn open_orders_after(
-    ex: &mut PgConnection,
-    after_block: i64,
+pub fn open_orders_by_time_and_uids<'a>(
```

Comment:
nit: this function is called `_by_time_and_uids`, but the query is by time OR uid? which premise is the correct one?